tests/searchcommands/test_internals_v1.py

```diff
@@ -225,53 +232,56 @@ def test_input_header(self):
 
         input_header = InputHeader()
 
-        with closing(StringIO('\r\n'.encode())) as input_file:
+        print("String: %s" % type('\r\n'))
+        print("Encode: %s" % type('\r\n'.encode()))
+
```

Comment:
just for debuging? remove these two lines

Reply:
Good call, will remove it.